### DISCUSSION TITLE
Is there any reason that this is not a Nuxt project?

### DISCUSSION THREAD

**Discussion by babakfp** [vuejs/vitepress#3826] [Q&A] [answered]
Hi 👋

I was wondering why VitePress doesn't use Nuxt. Are there any limitations or a reason not to? What benefits will VitePress get if it switches to Nuxt?

**Comment by brc-dd** [comment, marked as answer]
There are already Nuxt themes like Docus that use mdc syntax. You can probably use them if you want more features. VitePress is intentionally meant to be smaller and faster than full fledged meta frameworks like Nuxt. We won’t be switching to Nuxt at least in near future. However there are efforts to unify some of the internal tooling so that both Nuxt and VitePress can use similar transforms.
